CHANGELOG.md

```diff
@@ -8,43 +8,82 @@ A breaking change will get clearly marked in this log.
 
 ## [v9.0.0](https://github.com/stellar/js-stellar-sdk/compare/v8.2.5...v9.0.0)
 
-### Add
+This release adds **support for Automated Market Makers**. For details, you can refer to [CAP-38](https://stellar.org/protocol/cap-38) for XDR changes and [this document](https://docs.google.com/document/d/1pXL8kr1a2vfYSap9T67R-g72B_WWbaE1YsLMa04OgoU/view) for detailed changes to the Horizon API.
+
 
-- Introduced a `LiquidityPoolCallBuilder` to make calls to the new `/liquidity_pools` endpoint, featuring:
-  - filtering by reserve asset via `?reserves=[...]` ([#682](https://github.com/stellar/js-stellar-sdk/pull/682)), and 
-  - retrieving a specific pool via `/<id>/` ([#687](https://github.com/stellar/js-stellar-sdk/pull/687)).
+### Add
 
-- Expands the `Operation`, `Transaction`, and `Effects` call builders to allow querying for a specific liquidity pool by ID ([#689](https://github.com/stellar/js-stellar-sdk/pull/689)).
-- New effect types: `DepositLiquidityEffect`, `WithdrawLiquidityEffect`, `LiquidityPoolTradeEffect`, `LiquidityPoolCreatedEffect`, `LiquidityPoolRemovedEffect` and `LiquidityPoolRevokedEffect` ([#690](https://github.com/stellar/js-stellar-sdk/pull/690)).
-- The `RevokeSponsorshipOperationResponse` interface can now contain an optional attribute `trustline_liquidity_pool_id` for when a liquidity pool trustline is revoked.
-- Add operations `DepositLiquidityOperationResponse` and `WithdrawLiquidityOperationResponse`  ([#692](https://github.com/stellar/js-stellar-sdk/pull/692)).
+- Introduced a `LiquidityPoolCallBuilder` to make calls to a new endpoint:
+  * `/liquidity_pools[?reserves=...]` - a collection of liquidity pools, optionally filtered by one or more assets ([#682](https://github.com/stellar/js-stellar-sdk/pull/682))
+  * `/liquidity_pools/:id` - a specific liquidity pool ([#687](https://github.com/stellar/js-stellar-sdk/pull/687))
+
+- Expanded the `TransactionCallBuilder`, `OperationCallBuilder`, and `EffectsCallBuilder`s to apply to specific liquidity pools ([#689](https://github.com/stellar/js-stellar-sdk/pull/689)). This corresponds to the following new endpoints:
+  * `/liquidity_pools/:id/transactions`
+  * `/liquidity_pools/:id/operations`
+  * `/liquidity_pools/:id/effects`
+
+- Expanded the `TradesCallBuilder` to support making liquidity pool trades and accepts a new `trade_type` filter ([#685](https://github.com/stellar/js-stellar-sdk/pull/685)):
```

Comment:
👀  The `TradesCallBuilder` doesn't support _making_ trades but _fetching_ trades, right?

```suggestion
- Expanded the `TradesCallBuilder` to support fetching liquidity pool trades and accepts a new `trade_type` filter ([#685](https://github.com/stellar/js-stellar-sdk/pull/685)):
```

Reply:
Excellent point!

Reply:
Excellent point!